src/watcher/link.ts

```diff
@@ -90,6 +90,8 @@ export class Link {
     this.childProcess = nodecp.fork(require.resolve('./runner'), [], {
       cwd: process.cwd(),
       stdio: 'pipe',
+      // --experimental-worker for Node 10
+      execArgv: [...process.execArgv, '--experimental-worker'],
```

Comment:
Final solution, its a PITA to try and manage this at the root shebang level. We'll see how it goes...